install/overlays/libvirt/kustomization.yaml

```diff
@@ -20,6 +20,9 @@ configMapGenerator:
   - LIBVIRT_URI="qemu+ssh://root@192.168.122.1/system?no_verify=1" #set
   - LIBVIRT_NET="default" # set
   - LIBVIRT_POOL="default" # set
+  #- LIBVIRT_LAUNCH_SECURITY="" #sev or s390-pv
+  #- DISABLECVM="" #Uncomment and set if you want a generic VM
+  #- LIBVIRT_FIRMWARE="" # Default /usr/share/edk2/ovmf/OVMF_CODE.fd
```

Comment:
`# Uncomment and set if you want to change the firmware path. Defaults to /usr/share/edk2/ovmf/OVMF_CODE.fd`